app/components/sdg/targets/plain_tag_list_component.rb

```diff
@@ -0,0 +1,36 @@
+class SDG::Targets::PlainTagListComponent < ApplicationComponent
+  include SDG::TagList
+
+  private
+
+    def record
+      record_or_name
+    end
+
+    def tags
+      targets.sort[0..(limit.to_i - 1)]
+    end
+
+    def text(target)
+      "#{SDG::Target.model_name.human} #{target.code}"
+    end
+
+    def see_more_link
+      count = count_out_of_limit(targets)
+      link_to "#{count}+", polymorphic_path(record),
+              class: "more-#{i18n_namespace}",
+              title: t("sdg.#{i18n_namespace}.filter.more", count: count)
+    end
+
+    def targets
+      record.sdg_targets
+    end
+
+    def limit?
+      count_out_of_limit(targets) > 0
```

Comment:
In this case I like to use the `.positive?` method
https://apidock.com/rails/Numeric/positive%3F

Reply:
Thanks @gabrielbaldao, I applied your suggestion.